src/ui/Analytics/LiveAnalyticsClient.ts

```diff
@@ -174,8 +175,13 @@ export class LiveAnalyticsClient implements IAnalyticsClient {
     var customEvent = this.buildCustomEvent(actionCause, metaObject, element);
     this.triggerChangeAnalyticsCustomData('CustomEvent', metaObject, customEvent);
     this.checkToSendAnyPendingSearchAsYouType(actionCause);
+    var convertedCustomEvent = APIAnalyticsBuilder.convertCustomEventToAPI(customEvent);
     $$(this.rootElement).trigger(AnalyticsEvents.customEvent, <IAnalyticsCustomEventArgs>{
-      customEvent: APIAnalyticsBuilder.convertCustomEventToAPI(customEvent)
+      customEvent: convertedCustomEvent
+    });
+    $$(this.rootElement).trigger(AnalyticsEvents.coveoUAEventReady, <ICoveoUAEventArgs>{
```

Comment:
I left a similar comment on another PR related to using `UA` in the name. I just want to make sure the naming is consistent within the scope of the project. Do we normally use `UA` or `Analytics` or something else?

I find it's a little tidier when the naming is consistent :)

Reply:
I think we typically use `Analytics`.

I'll also remove the `Coveo` prefix

Reply:
(I'll keep the `coveo` prefix for the IAnalyticsEventArgs `coveoAnalyticsEventData` property however, since it populates the dataLayer, we'll want to make sure it's properly scoped to reduce the odds of it overriding other stuff).